### PR TITLE
Changes to the topic CLI command

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview:
https://niamh.preview.docs.r3.com/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.html
https://niamh.preview.docs.r3.com/en/platform/corda/5.1/reference/corda-cli/topic.html

### CODE REVIEW

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 1]
```diff
@@ -82,101 +97,100 @@ corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 1
...
 
-Where `<FILE>` is the name of the file in which to save the script and `<CONCURRENCY>` is the number of topics to create in parallel to speed execution.
+2. Review the output, make any necessary changes, and save the configuration to a file.
+3. Use the `connect` sub-command with the `-f` argument to connect to the Kafka broker and execute the configuration:
 
-For example:
+   {{< tabs name="create-topics">}}
+   {{% tab name="Bash" %}}
+   ```sh
+   corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> -f <YAML-TOPIC-FILE> create -r <REPLICAS> -p <PARTITIONS> connect
```
Comment (bpaunescu) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md:140]: The -f (--file) argument belongs to the `connect` sub-command. I think it should follow it in the example. 
```suggestion
   corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> create -r <REPLICAS> -p <PARTITIONS> connect -f <YAML-TOPIC-FILE>
```

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 2]
```diff
@@ -82,101 +97,100 @@ corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 1
...
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+   ```shell
+   corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> -f <YAML-TOPIC-FILE> create -r <REPLICAS> -p <PARTITIONS> connect
```
Comment (bpaunescu) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md:145]:
```suggestion
   corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> create -r <REPLICAS> -p <PARTITIONS> connect -f <YAML-TOPIC-FILE> 
```

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 3]
```diff
@@ -82,101 +97,100 @@ corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 1
...
-```
-{{% /tab %}}
-{{< /tabs >}}
+   For example:
 
-If you are authenticating Kafka users, the Corda CLI can also create Access Control List (ACL) entries as appropriate for each Corda worker.
+   {{< tabs name="create-topics-example">}}
+   {{% tab name="Bash" %}}
+   ```sh
+   corda-cli.sh topic -b kafka-1.example.com -k config.properties -f topics.yaml create -r 3 -p 10 connect
```
Comment (bpaunescu) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md:155]:
```suggestion
   corda-cli.sh topic -b kafka-1.example.com -k config.properties create -r 3 -p 10 connect -f topics.yaml
```

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 4]
```diff
@@ -82,101 +97,100 @@ corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 1
...
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+   ```shell
+   corda-cli.cmd topic -b kafka-1.example.com -k config.properties -f topics.yaml create -r 3 -p 10 connect
```
Comment (bpaunescu) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md:160]:
```suggestion
   corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 10 connect -f topics.yaml
```

File: content/en/platform/corda/5.1/reference/corda-cli/topic.md [hunk 5]
```diff
@@ -21,26 +21,38 @@ table th:nth-of-type(2) {
...
 
 ## create
 
+The following table lists the `create` sub-commands and arguments:
+
+| Argument           | Description                                                                                                                                                                          |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| -r, \-\-replicas   | The number of replicas.                                                                                                                                                              |
+| -p, \-\-partitions | The number of partitions.                                                                                                                                                            |
+| -t, \-\-tag        | One or more tags associated with topics and their respective number of partitions. For example:<br> `-t t01=partitions:3 -t t02=partitions:599999`.         |
```
Comment (bpaunescu) [inline content/en/platform/corda/5.1/reference/corda-cli/topic.md:39]: @davidcurrie I know this `tag` arg was added for internal use. Should we advertise this in the docs?

**Review by bpaunescu — CHANGES_REQUESTED** [review]
Only a couple of changes needed. And @davidcurrie could advise on keeping or hiding the tag arguments.

File: content/en/platform/corda/5.1/reference/corda-cli/topic.md (hunk 5 above)
Comment (davidcurrie) [reply content/en/platform/corda/5.1/reference/corda-cli/topic.md:39]: Actually, the `tag` argument was not implemented at all. Instead, the approach was to allow an override file to be specified, which has the same format as the generated file. See the [PR description](https://github.com/corda/corda-runtime-os/pull/4857#issue-1940792518) for more details. I think it's fine to document the existence of the argument but I wouldn't go into any detail on its usage or include it in any of the examples.

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**